src/contract.rs

```diff
@@ -1120,7 +1120,7 @@ fn modify_contract(
         bid_required_attributes,
     )?;
 
-    let response = Response::new();
+    let response = Response::new().add_attribute("action", "modify_contract");
```

Comment:
Are there potentially other `"action"` values that could be put there? It might be worthwhile to define some kind of enum with these actions as variants and convert it to a string when building `Response`.

Reply:
i think we could use an enum, right now we are using `action` in the response to easily identity the type of contract execution that took place when consuming blockchain data. `create_ask`, `create_bid`, ... etc. all have their own `action` defined via a hardcoded string right now (could be converted to an enum)

I could try to update them all